futures/src/tests.rs

```diff
@@ -19,18 +19,22 @@ fn qos_and_queue() {
             handle.spawn(heartbeat_future_fn(&heartbeat_client).map_err(|_| ()));
             client.create_channel()
         }).and_then(|channel| {
-            channel.queue_declare("qos_and_queue", &Default::default(), &FieldTable::new())
-                .and_then(move |_| {
-                    channel.basic_publish(
-                        "",
-                        "qos_and_queue",
-                        b"hello from lapin_futures",
-                        &Default::default(),
-                        Default::default())
-                        .map(move |_| channel)
-                })
-                .and_then(move |channel| {
-                    channel.queue_delete("qos_and_queue", &Default::default())
+            channel.basic_qos(&BasicQosOption { prefetch_count: 16, ..Default::default() })
```

Comment:
A whole new test doesn't seem necessary, just adding the call in the existing test should be enough